Extensions/RepeatEveryXSeconds.json

```diff
@@ -0,0 +1,684 @@
+                  "type": {
+                    "inverted": false,
+                    "value": "Egal"
+                  },
+                  "parameters": [
+                    "GetArgumentAsNumber(\"limit\")",
+                    "=",
+                    "-1"
+                  ],
+                  "subInstructions": []
+                },
+                {
+                  "type": {
+                    "inverted": false,
+                    "value": "VarScene"
+                  },
+                  "parameters": [
+                    "__RepeatEveryXSeconds_.Counter[GetArgumentAsString(\"timerName\")]",
+                    "<",
+                    "GetArgumentAsNumber(\"limit\")"
+                  ],
+                  "subInstructions": []
+                }
+              ]
+            }
+          ],
+          "actions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "ResetTimer"
+              },
+              "parameters": [
+                "player",
+                "GetArgumentAsString(\"timerName\")"
+              ],
+              "subInstructions": []
+            },
+            {
+              "type": {
+                "inverted": false,
+                "value": "ModVarScene"
+              },
+              "parameters": [
+                "__RepeatEveryXSeconds_.Counter[GetArgumentAsString(\"timerName\")]",
+                "+",
+                "1"
+              ],
+              "subInstructions": []
+            },
+            {
+              "type": {
+                "inverted": false,
+                "value": "SetReturnBoolean"
+              },
+              "parameters": [
+                "True"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "events": []
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The name of the timer to loop",
+          "longDescription": "",
+          "name": "timerName",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The time in seconds between each trigger",
+          "longDescription": "",
+          "name": "time",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The limit of loops",
+          "longDescription": "The condition will trigger at most this amount of times. Make it -1 for looping forever.",
+          "name": "limit",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    }
+  ],
+  "eventsBasedBehaviors": [
+    {
```

Comment:
Can you explain the advantage of the behavior compared to a set of actions? This avoids to repeat the name of the timer I guess?

Reply:
> This avoids to repeat the name of the timer I guess?

Yes, that's it. If that is not a good idea though i can make those regular actions.

Reply:
I'm not sure tbh, the only real risk for now is that behaviors can't be easily renamed in objects.